docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
+cluster. Moving nodes to a new cluster is particularly useful when the voting
+configuration of a node has been reset, and a new cluster is bootstrapped.
 
 .. WARNING::
 
-  Execution of this command can lead to arbitrary data loss. Only run this tool
-  if you understand and accept the possible consequences and have exhausted all
-  other possibilities for recovery of your cluster.
+  Moving nodes between clusters can lead to arbitrary data loss. Different
+  clusters can have utterly different cluster metadata, and there is no way to
+  merge the metadata of two clusters safely.
+
+Each cluster creates a unique identifier at its initial start-up to protect
+yourself against inadvertently joining the wrong clusters. This unique
+identifier is known as the cluster UUID (Universal Unique ID). Every node
+records the UUID of its cluster and refuses to join a cluster with a different
+UUID.
+
+However, if the cluster of a node permanently fails then you may want to move
+the node into a new cluster.
+
+To reset the cluster UUID of a node and detach the node from its cluster,
+give this command:
+
+``crate-node detach-cluster``
+
+The node can now join another cluster with a different UUID.
+
+For example, after unsafe cluster bootstrapping you need to detach all other
+surviving nodes from their old cluster so that they can join the new,
+unsafely-bootstrapped cluster.
+
+.. WARNING::
 
-The sequence of operations for using this tool are as follows:
+  Executing this command can lead to arbitrary data loss. Run this tool only
+  if you understand and accept the possible consequences. Also only run it if
+  you have exhausted all other possibilities for the recovery of your cluster.
 
-* Make sure you have really lost access to every one of the master-eligible nodes
-  in the cluster, and they cannot be repaired or recovered by moving their data
-  paths to healthy hardware.
+Follow this sequence before using the detach-cluster tool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Start a new cluster and verify that it is healthy. This cluster may comprise
-  one or more brand-new master-eligible nodes, or may be an unsafely-bootstrapped
-  cluster formed as described above.
+* Make entirely sure you have lost access to each of the master-eligible
+  nodes in the cluster.
+
+* Make entirely sure that the master-eligible nodes cannot be repaired or
+  recovered by moving their data paths to healthy hardware
+
+* Start a new cluster and verify that it is healthy. The cluster may comprise
+  one or more new master-eligible nodes, or maybe an unsafely-bootstrapped
+  cluster, formed in the section
+  :ref:`conf-env-re-purposing-nodes-unsafe-bootstrapping`
 
 * Stop all remaining data nodes.
 
-* On each data node, run the ``crate-node detach-cluster`` tool as shown
-  below. Verify that the tool reported Node was successfully detached from
-  the cluster.
+* On each data node, run the ``detach-cluster`` tool as demonstrated below.
```

Comment:
I prefer having the full command mentioned such as ``crate-node detach-cluster``.